app/src/forms/form/service.js

```diff
@@ -193,6 +230,8 @@ const service = {
       obj.ministry = data.ministry;
       obj.apiIntegration = data.apiIntegration;
       obj.useCase = data.useCase;
+      obj.showAssigneeInSubmissionsTable = data.showAssigneeInSubmissionsTable || false;
```

Comment:
See my note above about not throwing exceptions in _validateAssigneeVisibilitySettings.

obj.showAssigneeInSubmissionsTable = _validateAssigneeVisibilitySettings(data); // function returns true or false.

Since we are not exactly validating, you can choose a better name (_setAssigneeInSubmissionsTable or something).